github_api_helper.js

```diff
@@ -43,12 +44,12 @@ async function getConfigFile (context) {
     return await context.github.repos.getContents({
       owner,
       repo,
-      path: config.configFilePath,
+      path: customConfigPath,
       headers: { accept: rawMediaType }
     })
   } catch (error) {
     if (error.code === 404) {
-      return undefined
+      return null
```

Comment:
OOI we had a discussion thread on the first patch in this PR where you concluded that `undefined` made more sense than `null` - what changed? Would this change be more reasonable as part of that first patch?

Reply:
Yes, that was a mistake. 
I fixed it.